.github/workflows/release.yml

```diff
@@ -103,6 +104,35 @@ jobs:
           fi
           git tag -l -n --format="%(contents:subject)" $(git describe --tags --abbrev=0) > TAG-COMMENT.txt
 
+      - name: MSI
+        shell: bash
+        if: matrix.os == 'windows-latest'
+        run: |        
+          # Return a qldb.exe
+          cargo fetch && cargo build --release  
+          
+          strip "target/${{ matrix.target }}/release/${{ matrix.bin }}"
+          
+          cargo install cargo-wix
+          
+          # We have now a file in `target/wix/main.wxs`
+          # that has the script for making an MSI installe
+               
+          # Create MSI based on target/wix/main.wxs
```

Comment:
Right now the custom wix file is under amazon-qldb-shell/wix directory but cargo wix is expecting a wix file under amazon-qldb-shell/target/wix

Reply:
Yes, that's how it works.  Is there something we need to change?

Reply:
How is this running successfully if we are placing the wxs file in the wrong directory?

Reply:
The wxs file is in the right directory amazon-qldb-shell/wix.  The command cargo wix will take the wxs file and output the msi file into amazon-qldb-shell/target/wix.  Then we move and rename the msi from target/wix into the amazon-qldb-shell directory.